decentraland/unity-explorer
# chore: update smr only when visible

## What does this PR change?

Set the `updateWhenOffscreen` flag to `false` for SMR. In GP it was creating an unnecessary overhead

It goes from this

![image](https://github.com/user-attachments/assets/4c3e5d19-d802-44d8-b49a-ff37c819001d)

to this

![image](https://github.com/user-attachments/assets/d1f94127-3e94-43ab-9811-60c5afae4afa)

440 SMR renderers where updated unnecessarily just by standing on the clouds in GP

## How to test the changes?

1. Launch the explorer
2. Walk around for GP and look for robots/characters. They should all behave normally
3. Check the shark in Goerli at `73,-8`. Check that it swims, and bites when clicked

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
